app/globals.css

```diff
@@ -9,21 +9,21 @@
 
     --offset: #f5f6f8;
 
-    --accent: #27AE60;
+    --accent: #27ae60;
     --accent-foreground: #f4f4f5;
-    --accent-subdued: #E9F7EF;
+    --accent-subdued: #e9f7ef;
 
-    --primary: #353A44;
+    --primary: #27ae60;
```

Comment:
just a heads up that this changes some of the colors on the dashboard pages! i've been previously using `accent` for the green color

![CleanShot 2024-05-01 at 00 38 04@2x](https://github.com/stripe/stripe-connect-furever-demo/assets/99840257/1ab5e822-6e70-47b5-bbfa-197fec326dbc)

Reply:
changed to using accent!